appveyor.yml

```diff
@@ -0,0 +1,13 @@
+version: 1.0.{build}
+image: Visual Studio 2017
+configuration: 
+  - Debug
+  - Release
+#cache:
```

Comment:
Do you need cache in the appveyor.yml? If yes then uncomment it, otherwise delete it, please.